keras_spiking/layers.py

```diff
@@ -21,7 +23,7 @@ class KerasSpikingCell(tf.keras.layers.Layer):
     state_size : int
         Dimensionality of cell state.
     dt : float
-        Length of time (in seconds) represented by one time step.
+        Length of time (in seconds) represented by one time step. Defaults to 0.001s.
```

Comment:
Do we want to reference `keras_spiking.default.dt` here? Otherwise, users might not know it exists.